sandbox/libs/authn/build.gradle

```diff
@@ -18,7 +18,7 @@ dependencies {
   implementation "com.fasterxml.jackson.core:jackson-annotations:${versions.jackson}"
   implementation "com.fasterxml.jackson.core:jackson-databind:${versions.jackson_databind}"
 
-  implementation 'org.apache.shiro:shiro-core:1.9.1'
+  api 'org.apache.shiro:shiro-core:1.9.1'
```

Comment:
This will let you access internal dependencies of shiro, is that the intention?

Reply:
yes